src/components/DragAndDrop.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Box, Center, Text, Image } from "@chakra-ui/react";
+import FileUploadButton from "./FileUploadButton";
+
+type DrapAndDropProps = {
+    setFiles: (files: File[]) => void;
+};
+
+// TODO could be a lot of refactor for actual drag and drop
+// TODO detect mobile users and don't show d&d
+const DragAndDrop: React.FC<DrapAndDropProps> = ({ setFiles }): JSX.Element => {
+    return (
+        <Box
+            width="500px"
+            border="2px"
+            borderStyle="dashed"
+            borderColor="gray.500"
+        >
+            <Center>
+                {/* TODO use svg and local image */}
+                <Image
+                    src="http://www.clker.com/cliparts/S/j/7/o/b/H/cloud-upload-outline.svg.thumb.png"
+                    marginTop="20px"
+                ></Image>
+            </Center>
+            <Center>
+                <Text
+                    fontWeight="400"
+                    fontSize="18px"
+                    mt="18px"
+                    marginTop="20px"
```

Comment:
mt == marginTop. I would try to use mt in the future.
```suggestion
                    marginTop="20px"
```